modules/manually-configure-iam-nutanix.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/configuring-iam-nutanix.adoc
+
+:_content-type: PROCEDURE
+[id="manually-create-iam-nutanix_{context}"]
+= Configuring IAM for Nutanix
+
+Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installation program configures the CCO for manual mode, you must specify the identity and access management secrets.
+
+.Prerequisites
+
+* You have configured the `ccoctl` binary.
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Create a YAML file that contains the credentials data in the following format:
++
+.Credentials data format
+[source,yaml]
+----
+credentials:
+- type: basic_auth <1>
+  data:
+    prismCentral: <2>
+      username: <username_for_prism_central>
+      password: <password_for_prism_central>
+    prismElements: <3>
+    - name: <name_of_prism_element>
+      username: <username_for_prism_element>
+      password: <password_for_prism_element>
+----
+<1> Specify the authentication type. Only basic authentication is supported.
+<2> Specify the Prism Central credentials.
+<3> Optional: Specify the Prism Element credentials.
+
+. Extract the list of `CredentialsRequest` custom resources (CRs) from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc adm release extract --credentials-requests --cloud=nutanix \
+--to=<path_to_directory_with_list_of_credentials_requests>/credrequests \ <1>
+quay.io/<path_to>/ocp-release:<version>
+----
++
+<1> Specify the path to the directory that contains the files for the component `CredentialsRequests` objects.
```

Comment:
This may need update on note 1.
Refer to https://bugzilla.redhat.com/show_bug.cgi?id=2097051
In which note 1 is "credrequests is the directory where the list of CredentialsRequest objects is stored. This command creates the directory if it does not exist."

Reply:
Done. Thank you.